CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Change Log
 
-### 0.1.0
+## 0.81 (August 27, 2021)
+
+### New Feature
```

Comment:
Would it be helpful to demonstrate the cell reuse problem with a diagram or graphic? To keep things simple (for myself at least...) I referred to the code.

Reply:
I think without diagram is ok since we can refer to the open source code and also the interface for supporting the feature is quite clean and simple.